src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import FakeBookings from "./data/fakeBookings.json";
+import moment from "moment/moment";
+
+const SearchResults = (props) => {
```

Comment:
This looks great!
You did a good job of passing the props in and using them in the component 👍